src/main/java/bio/terra/cli/command/resource/Delete.java

```diff
@@ -22,6 +23,10 @@ protected void execute() {
 
     Resource resourceToDelete = Context.requireWorkspace().getResource(resourceNameOption.name);
 
+    if (resourceToDelete.getResourceType().equals(Resource.Type.DATAPROC_CLUSTER)) {
+      CommandUtils.checkDataprocSupport();
```

Comment:
Flag - vwb__cli_token_refresh_enabled seems to be targetted only for CLI. Is it possible to create a data proc cluster on UI, and then be unable to access it on CLI? 
This might not be a blocker if behavior is expected and informed to solutions team. 

 further -
This check prevents all operations on the cluster, but does not hide it. 
Delete is not allowed even if the cluster was created on UI. 
what is the expected behavior when user tries to delete the workspace? will it fail?

Reply:
Flag - yes that's possible, we have separate flags for UI and CLI, but they should be synced and toggled together.